bdanielle092/HomeMade
# Fm recipe cards

# Description
I have to login/logout/register working. I have added text area to the recipe cards. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
I test login, logout, register. I also check the detail cards to see if the recipes are in there.
# Checklist:
